src/ppscore/calculation.py

```diff
@@ -29,21 +34,18 @@
 NUMERIC_AS_CATEGORIC_BREAKPOINT = 15
 
 
-def _calculate_model_cv_score_(df, target, feature, metric, model, **kwargs):
+def _calculate_model_cv_score_(df, target, feature, metric, model, cv, **kwargs):
     "Calculates the mean model score based on cross-validation"
     # Sources about the used methods:
     # https://scikit-learn.org/stable/modules/tree.html
     # https://scikit-learn.org/stable/modules/cross_validation.html
     # https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.cross_val_score.html
 
-    # shuffle the rows - this is important for crossvalidation
-    # because the crossvalidation just takes the first n lines
-    # if there is a strong pattern in the rows eg 0,0,0,0,1,1,1,1
-    # then this will lead to problems because the first cv sees mostly 0 and the later 1
-    # this approach might be wrong for timeseries because it might leak information
-    df = df.sample(frac=1, random_state=RANDOM_SEED, replace=False)
```

Comment:
In case that we use the default CV, I think that we still need this

Reply:
Yeah I think I meant to bring it back and forgot..

Reply:
Alright, no worries :)